contrib/world_builder/CMakeLists.txt

```diff
@@ -433,18 +433,15 @@ if (NOT MSVC AND NOT APPLE)
    endif()
    IF ( CMAKE_BUILD_TYPE STREQUAL Coverage )
      if(NOT ${CMAKE_VERSION} VERSION_LESS "3.13.0") # Preventing issues with older cmake compilers which do not support VERSION_GREATER_EQUAL
-     SET(WB_LINKER_OPTIONS -lstdc++ --coverage -fprofile-arcs -ftest-coverage)
+     SET(WB_LINKER_OPTIONS --coverage -fprofile-arcs -ftest-coverage)
      else()
-      SET(WB_LINKER_OPTIONS "-lstdc++ --coverage -fprofile-arcs -ftest-coverage")
+      SET(WB_LINKER_OPTIONS "--coverage -fprofile-arcs -ftest-coverage")
      endif()
-   else()
-     SET(WB_LINKER_OPTIONS -lstdc++)
    endif()
 
    SET(WB_VISU_LINKER_OPTIONS "-pthread")
 elseif(APPLE)
-    SET(WB_COMPILER_OPTIONS_PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-std=c++14 -Wpointer-arith -Wwrite-strings -Wsynth -Wsign-compare -Woverloaded-virtual -Wno-literal-range -Wno-parentheses -Wno-unused-local-typedefs -Wcast-qual -fstrict-aliasing -stdlib=libc++ -Wuninitialized -Werror=uninitialized -Wdangling-else>)
-    SET(WB_LINKER_OPTIONS -lc++)
+    SET(WB_COMPILER_OPTIONS_PRIVATE $<$<COMPILE_LANGUAGE:CXX>:-std=c++14 -Wpointer-arith -Wwrite-strings -Wsynth -Wsign-compare -Woverloaded-virtual -Wno-literal-range -Wno-parentheses -Wno-unused-local-typedefs -Wcast-qual -fstrict-aliasing -Wuninitialized -Werror=uninitialized -Wdangling-else>)
```

Comment:
Setting the standard to c++14 unconditionally is also problematic because we might be compiling deal.II/ASPECT with a different standard.

Reply:
Yes, indeed. You should be using the cmake facilities to set the language standard, like
```
SET (CMAKE_CXX_STANDARD 20)
SET (CMAKE_CXX_STANDARD_REQUIRED ON)
SET (CMAKE_CXX_EXTENSIONS OFF)
```
There are ways to make this target-dependent, which is probably the better choice.

Reply:
But all of that can also happen in a separate patch.

Reply:
apple clang is defaulting to c++-14 anyways, I think.